xicam/Acquire/controlwidgets/typhosmotorcontroller.py

```diff
@@ -0,0 +1,12 @@
+from xicam.plugins import ControllerPlugin
+from typhos import TyphosDeviceDisplay
+import typhos
+
+
+class TyphosMotorController(TyphosDeviceDisplay, ControllerPlugin):
```

Comment:
rename -> TyphosController
check for controller_class field in happi config to decide which controller to use for display